lib/writer.js

```diff
@@ -147,3 +162,44 @@ function currentDate() {
 
   return util.format('%d-%s-%s', now.getFullYear(), pad(now.getMonth() + 1), pad(now.getDate()));
 }
+
+function nonGithubUrl(url) {
```

Comment:
Pretty sure https://github.com/npm/normalize-package-data#what-normalization-currently-entails normalizes this already.